### PR TITLE
ROCm support for sglang.check_env

### PR DESCRIPTION
**Pull request by hliuca**
The check_env module supports CUDA only. This PR enables ROCm support, which has been tested on MI300X and H100.

image

image

### CODE REVIEW

File: python/sglang/check_env.py [hunk 1]
```diff
@@ -9,6 +9,19 @@
 
 import torch
 
+is_rocm = False
```
Comment (merrymercy) [inline python/sglang/check_env.py]: use this one?
https://github.com/sgl-project/sglang/blob/ece724910afd624e8aac85444472440d6586c3e4/python/sglang/srt/utils.py#L67-L69

File: python/sglang/check_env.py [hunk 2]
```diff
@@ -63,13 +76,22 @@ def get_cuda_info():
     """
     Get CUDA-related information if available.
     """
-    cuda_info = {"CUDA available": torch.cuda.is_available()}
+    if is_rocm:
```
Comment (merrymercy) [inline python/sglang/check_env.py]: Make cuda as the first branch.

**Review by merrymercy — CHANGES_REQUESTED** [review]

**Comment by merrymercy** [comment]
also, please fix the lint https://sgl-project.github.io/references/contributor_guide.html#format-your-code

**Comment by hliuca** [comment]
Thank you @merrymercy. I will fix those according to your comments.

File: python/sglang/check_env.py (hunk 2 above)
Comment (hliuca) [reply python/sglang/check_env.py]: the is_cuda returns true even on HIP :-)

**Comment by hliuca** [comment]
The issues have been addressed. Thank you @merrymercy

**Pull request merged**